src/stubs/ErrorUtils.js

```diff
@@ -11,13 +11,17 @@
 
 /* jslint unused:false */
 
-var ErrorUtils = {
-  applyWithGuard(callback, context, args, onError, name) {
-    return callback.apply(context, args);
-  },
-  guard(callback, name) {
-    return callback;
-  },
-};
+if (global.ErrorUtils) {
```

Comment:
@zpao @yungsters Let me know if I need to update this...admittedly, I'm kind of working blind when it comes to how these changes affect the internal FB codebase.